### ISSUE TITLE
uSync not appearing in the backoffice (Umbraco 15 on Ubuntu)

### ISSUE THREAD

**Issue by stvnhrlnd** [KevinJump/uSync8#683]
**Describe the bug**
The Synchronisation menu does not appear in the Settings section on a clean install of Umbraco 15.0.0 with uSync 15.0.0 on Ubuntu 24.04.1 with .NET SDK 9.0.100.

The browser console reports a 404 on `usync.js`, but I've noticed that `uSync.js` _does_ exist, so this looks to be an issue of case sensitivity (and likely not affecting Windows installs). See screenshots below.

**To Reproduce**
1. Perform a clean install of Umbraco and uSync using [this script](https://psw.codeshare.co.uk/?TemplateName=Umbraco.Templates&TemplateVersion=15.0.0&Packages=uSync%7C15.0.0&UserEmail=admin%40example.com&ProjectName=MyProject&CreateSolutionFile=false&SolutionName=MySolution&UseUnattendedInstall=true&DatabaseType=SQLite&UserPassword=1234567890&UserFriendlyName=Administrator&IncludeStarterKit=false&StarterKitPackage=clean&OnelinerOutput=false&RemoveComments=false).
2. Log in to the backoffice.
3. Navigate to the Settings section.
4. Notice that the usual uSync Synchronisation menu is not there.

**Expected behavior**
The Synchronisation menu and other uSync backoffice extensions should appear.

**Screenshots**
404 on `usync.js`:
![Screenshot from 2024-11-15 10-14-30](https://github.com/user-attachments/assets/61e61b2d-a8d2-4676-8a17-7dd6bcb0fa8d)
But `uSync.js` is there:
![Screenshot from 2024-11-15 10-56-38](https://github.com/user-attachments/assets/19711eab-219b-4b05-9d74-2a13b277caa7)

**About your Site (please complete the following information):**
 - Umbraco Version: 15.0.0
 - uSync Version: 15.0.0
 - Browser: Chrome 131.0.6778.69, Firefox 132.0.2
 - Operating System: Ubuntu 24.04.1 LTS (running under Multipass)
 - .NET SDK Version: 9.0.100

**Comment by stvnhrlnd** [comment]
Temporarily working around this with a URL rewrite in `Program.cs`:

```csharp
app.UseRewriter(new RewriteOptions()
    .AddRewrite(
        @"^App_Plugins/uSync/usync.js(.*)",
        "App_Plugins/uSync/uSync.js$1",
        skipRemainingRules: true));
```

**Comment by KevinJump** [comment]
Fix avalible from our "nightly" feed. 

https://dev.azure.com/jumoo/Public/_artifacts/feed/nightly/NuGet/uSync/overview/15.0.1-build.20241115.1

We will just hold off a day or two encase anyone else finds an new issues and then push this as a release.

**Comment by stvnhrlnd** [comment]
Awesome. Thanks for the quick response!

**Comment by KevinJump** [comment]
Fixed in 15.0.1. https://www.nuget.org/packages/uSync/15.0.1

**Comment by stvnhrlnd** [comment]
Confirmed working, thanks again :)
